### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by poojaranjan**
Update Readme for Statuses and description.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,14 +4,14 @@
 
 Ethereum Improvement Proposals (EIPs) describe standards for the Ethereum platform, including core protocol specifications, client APIs, and contract standards.
 
-**Before you initiate a pull request**, please read the [EIP-1](https://eips.ethereum.org/EIPS/eip-1) process document. Ideas should be thoroughly discussed prior to opening a pull request, such as on the [Ethereum Magicians forums](https://ethereum-magicians.org) or in a GitHub issue in this repository.
+**Before you initiate a pull request**, please read the [EIP-1](https://eips.ethereum.org/EIPS/eip-1) process document. Ideas should be thoroughly discussed prior to opening a pull request on the [Ethereum Magicians forums](https://ethereum-magicians.org).
 
 This repository tracks the ongoing status of EIPs. It contains:
 
-- [Draft](https://eips.ethereum.org/all#draft) proposals which intend to complete the EIP review process.
+- [Draft](https://eips.ethereum.org/all#draft) proposals which indicates author is still making changes and is not ready for review.
```
Comment (lightclient) [inline README.md:11]:
```suggestion
- [Draft](https://eips.ethereum.org/all#draft) proposals where the author(s) are still making changes and is not ready for review.
```

File: README.md [hunk 2]
```diff
@@ -4,14 +4,14 @@
...
+- [Review ](https://eips.ethereum.org/all#review) for proposals ready for EIP review process and author is ready for improvement suggestions.
```
Comment (lightclient) [inline README.md:12]:
```suggestion
- [Review ](https://eips.ethereum.org/all#review) for proposals ready for EIP review process and the author(s) are ready for improvement suggestions.
```

**Review by lightclient — COMMENTED** [review]
Just some thoughts.

**Comment by poojaranjan** [comment]
@lightclient 
Looks like something went wrong on commits with the branch name.  Creating another pull request to update the suggestions.

Closing this PR in favor of #3721

**Pull request closed without merging**